.travis.yml

```diff
@@ -19,7 +19,7 @@ before_script:
 script:
   # - docker run -v $(pwd):/mnt koalaman/shellcheck **/*.sh(e[' [[ ! `echo "$REPLY" | grep "vendor/" ` ]]'])
   - diff -u <(echo -n) <(gofmt -d -s `find . -type f -name '*.go' -not -path "./vendor/*"`)
-  - diff -u <(echo -n) <(golint `go list ./... | grep -v /vendor/`)
+  - diff -u <(echo -n) <(golint `go list ./... | grep -v /vendor/ | grep -v /rtns/`)
```

Comment:
probably don't need to grep `rtns` anymore now that the golint complaint has been resolved 😁